src/main/java/io/fabric8/jenkins/openshiftsync/Constants.java

```diff
@@ -38,4 +38,8 @@
 	public static final String OPENSHIFT_SECRETS_TYPE_BASICAUTH = "kubernetes.io/basic-auth";
 	public static final String OPENSHIFT_SECRETS_TYPE_OPAQUE = "Opaque";
 	public static final String OPENSHIFT_BUILD_STATUS_FIELD = "status";
+
+    static final String LABEL_JENKINS = "jenkins";
```

Comment:
let's namespace this label.  `openshift.io/jenkins-credential-sync` and check for a value of `true` instead of `sync`.

Reply:
So we've hit a bit of a hiccup with making the label `openshift.io/jenkins-credential-sync`  ... the `/` in particular.

Can't just put that in the label as is, as http doesn't like slashes in its query params.

So I tried to utf-8 encode it, and then okhttp3 complains with this:

```
WARNING: Exec Failure: HTTP 400, Status: 400 - unable to parse requirement: invalid label key "openshift.io%2Fjenkins-credential-sync": name part must consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyName',  or 'my.name',  or '123-abc', regex used for validation is '([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9]')
java.net.ProtocolException: Expected HTTP 101 response but was '400 Bad Request'
	at okhttp3.internal.ws.RealWebSocket.checkResponse(RealWebSocket.java:216)
	at okhttp3.internal.ws.RealWebSocket$2.onResponse(RealWebSocket.java:183)
	at okhttp3.RealCall$AsyncCall.execute(RealCall.java:141)
	at okhttp3.internal.NamedRunnable.run(NamedRunnable.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

```

Evidently there has been some churn wrt this in the past with okhttp3, but was supposedly resolved.  For example, see https://stackoverflow.com/questions/30142626/how-to-add-query-parameters-to-a-http-get-request-by-okhttp ....following links off of that, it seemed to tolerate UTF-8 encoding

However our current combination of fabric and okhttp3 is not happy (and the okhttp3 pulled in by the fabric8 version sync plugin currently pulls in is greater than the version noted in the link above).

We could experiment with bumping fabric8 and by extension okhttp3 (we did so in the past but hit openshif version incompatibility issues as you may recall @bparees ... though in the "next release", with fixes backported to 3.6.1, we were planning on punting on older versions).

And I suppose there is the "personnel changes" wrt to fabric over the long haul as well.

All that said, I'm not sure I want to pull in the fabric version bump work load for this card / PR.

Or we no longer qualify our watch/query with the label, and then have to pull in all the secrets from the namespace(s), and then filter in the plugin.

Or we massage the label name to not have `/`.

@bparees ?

Reply:
ok. just call it "credential.sync.jenkins.openshift.io"